README.rst

```diff
@@ -44,6 +44,10 @@ DEMO
 ----
 GIF_.  Live_.
```

Comment:
hey it seems that somehow when you were rebasing you got some changes which werent yours.. could you fix this, or if you dont know how, ask on gitteR :)? Thanks!